extensions/cornerstone-dicom-seg/src/index.tsx

```diff
@@ -36,9 +38,15 @@ const extension = {
    */
   preRegistration: ({
     servicesManager,
-    commandsManager,
-    configuration = {},
-  }) => {},
+    extensionManager,
+  }: Types.Extensions.ExtensionParams): void => {
+    const {
+      toolGroupService
+    } = servicesManager.services;
+    // Create a tool group by default for use with this tool set.
+    createSEGToolGroupAndAddTools(toolGroupService, SEG_TOOLGROUP_DEFAULT, extensionManager);
```

Comment:
so we are doing create twice? since we already create inside the SEGViewport

```
toolGroup = createSEGToolGroupAndAddTools(
      toolGroupService,
      toolGroupId,
      extensionManager
    );
    ```